app/models/dish.rb

```diff
@@ -1,4 +1,10 @@
 class Dish < ApplicationRecord
   validates_presence_of :name, :description
   belongs_to :chef
+  has_many :dish_ingredients
+  has_many :ingredients, through: :dish_ingredients
+
+  def total_calories
```

Comment:
I didn't see a test for this method. Good job with writing it, just make sure to test it in the models.